### PR TITLE
Use build-tools-23.0.3 on Travis

### PR DESCRIPTION
**Pull request by jameswald**
See https://github.com/travis-ci/travis-ci/issues/5891 and https://github.com/travis-ci/travis-ci/issues/5648 regarding the `platform-tools` and `tools` additions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,4 +31,4 @@ env:
   - secure: mQwFIdOIooeZ1KkDhkhXdSORA3Tn713C7SYYaF/ZWhhhINHvwMjPAOT+R7zSyGGqjhpqklRbKmZXDJHGPdus1q5CvZLYvldeQXsEfzvav95aukHjNbOIa3y/5h1rZWXwTLR13aMv/o87SrTT2xiyE6XgxFxN3eSsu/zavrQGQkM=
```
Comment (akarnokd) [inline .travis.yml:31]: What's this?

Comment (akarnokd) [inline .travis.yml:31]: Oh, I see, just the line ending

**Comment by akarnokd** [comment]
It didn't really work, got a bunch of failures, but still there are [androidJavadocs](https://travis-ci.org/akarnokd/RxAgera/builds/125199942) failures (that I thought fixed in Android studio). Could you have a look at them?

**Comment by jameswald** [comment]
Sure I'll take a look. JDK 8 usually requires suppression of Javadoc warnings in Android projects. If that's the issue, I know where to grab a fix for that.

**Comment by akarnokd** [comment]
It seems to be that androidJavadocs doesn't pick up the dependencies properly, it misses the agera library for some reason. I had the same issue in Android Studio until I added [a few lines](https://github.com/akarnokd/RxAgera/blob/master/rxagera/build.gradle#L146) I found elsewhere. That's why I could then release to nexus from Studio via `uploadArchives`.

**Pull request merged**